build/stf-run-ci/tasks/create_catalog.yml

```diff
@@ -0,0 +1,145 @@
+---
+- name: Create service-telemetry-framework-index working directory
+  file:
+    path: working/service-telemetry-framework-index
+    state: directory
+    mode: '0755'
+
+- name: Create info variables from bundle generation output
+  set_fact:
+    sto_bundle_info: "{{ generate_bundle_sto.stdout }}"
+    sgo_bundle_info: "{{ generate_bundle_sgo.stdout }}"
+
+- name: Get the builder-dockercfg Secret name
+  command: oc get secret -n {{ namespace }} --field-selector='type==kubernetes.io/dockercfg' -ojsonpath='{.items[?(@.metadata.annotations.kubernetes\.io/service-account\.name=="builder")].metadata.name}'
+  register: secret_builder_dockercfg_name
+
+- name: Get contents of builder Secret
+  k8s_info:
+    api_version: v1
+    kind: Secret
+    name: "{{ secret_builder_dockercfg_name.stdout }}"
+    namespace: "{{ namespace }}"
+  register: secret_builder_dockercfg_results
+
+- name: Get builder-dockercfg authentication contents
+  set_fact:
+    builder_dockercfg_auth_results: "{{ secret_builder_dockercfg_results.resources[0].data['.dockercfg'] | b64decode }}"
+
+- name: Set internal registry authentication
+  set_fact:
+    internal_registry: "{{ builder_dockercfg_auth_results['image-registry.openshift-image-registry.svc:5000'] | to_json }}"
+
+- when: query('kubernetes.core.k8s', api_version='v1', kind='Secret', resource_name='service-telemetry-framework-index-dockercfg', namespace=namespace) | length == 0
+  block:
+  - name: Create config.json to import as Secret
+    template:
+      variable_start_string: "<<"
+      variable_end_string: ">>"
+      src: config-json.j2
+      dest: working/service-telemetry-framework-index/config.json
+
+  - name: Create a Secret for the dockercfg
+    command: oc create secret generic -n {{ namespace }} service-telemetry-framework-index-dockercfg --from-file=.dockerconfigjson=working/service-telemetry-framework-index/config.json --type=kubernetes.io/dockerconfigjson
+
+- name: Create ImageStream for ose-operator-registry
+  command: oc import-image -n {{ namespace }} ose-operator-registry:{{ default_operator_registry_image_tag }} --from={{ default_operator_registry_image_base }}:{{ default_operator_registry_image_tag }} --confirm
```

Comment:
I did a test of this locally to make sure I didn't cause any issues. Deployment/build works as expected.